app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
Like above, kind of think we should just be reusing existing `ApplicationUserRepresenter`. (since I think apps always want to talk in terms of application users -- unclaimed ones will just be missing most information)